Library/Homebrew/cmd/deps.rb

```diff
@@ -78,6 +81,7 @@ def deps
     end
 
     @only_installed_arg = args.installed? &&
+                          !args.send("1?") &&
```

Comment:
Presumably `args.1?` doesn't work. Could consider defining a new method `args.one?` which might be a bit nicer?

Reply:
I added to the existing `mode` structure so that each can check `mode.recursive?` instead.